t/tags.t

```diff
@@ -5,7 +5,7 @@ use Modern::Perl '2012';
 use utf8;
 
 use Test::More;
-use Log::Any::Adapter 'TAP', filter => "none";;
+#use Log::Any::Adapter 'TAP', filter => "none";;
```

Comment:
Why did you remove this?

Reply:
Just to remove some logs in the output while I was testing. I can put it back.